packages/graphql-codegen/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "baseUrl": "./src",
     "paths": {
       "joist-utils": ["../../utils/src"],
-      "joist-codgen": ["../../codegen/src"]
+      "joist-codegen": ["../../codegen/src"]
```

Comment:
@stephenh - I presume this was a typo? Let me know if there is some reasoning for this. It resulted in more diffs than I was expecting in the Codegen files.

Reply:
Oh wow, yep, good fix.